phpstan.neon.dist

```diff
@@ -1,5 +1,4 @@
 parameters:
-    phpVersion: 70100
```

Comment:
I'd keep the setting and bump it to `80100`.

Reply:
Updated this, but not sure its a good idea because its configured in two locations now, in the github ci yml setup and here. This might lead to missmatches when future changes are done.

Reply:
@greg0ire whats your opinion on this?

Reply:
In case of a mismatch, this setting wins. Setting the version explicitly in this config file should make the PHPStan result reproducible in a contributor's local setup. At least I would hope so. 😅 🤞

Reply:
The thing configured on Github action in the php version to install on the machine, whereas this represents the target version, I'd say these are 2 orthogonal settings, and when not specified, here, it will default to the current php version. When php 8.2 is going to come out, we still want to analyze the code as if it was written for 8.1, but might benefit from faster runs/more up to date deps by installing php 8.2. So I'd say it's a good practice to specify it.

Reply:
I'm not sure what happens if we have bits of code that uses syntax specific to 8.2 though :thinking: (like we have test code that uses `readonly` properties in the ORM).